build/ci/Dockerfile.unittest

```diff
@@ -25,6 +25,12 @@ RUN go get github.com/onsi/ginkgo/ginkgo@v1.16.4 \
     && make kustomize \
     && make controller-gen
 
+ARG YQ_VERSION=v4.13.5
+ARG BINARY=yq_linux_386
+RUN YQ_BINARY=$(if [ "$(uname -m)" = "x86_64" ]; then echo "yq_linux_amd64"; else echo "yq_linux_$(uname -m)" ; fi) \
+    && wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY} -O /usr/bin/yq \ 
```

Comment:
```suggestion
ARG BINARY
RUN YQ_ARCH=$(if [ "$(uname -m)" = "x86_64" ]; then echo "amd64"; else echo "$(uname -m)" ; fi) \
    && wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_${YQ_ARCH} -O /usr/bin/yq \ 
```

Reply:
done in this PR

Reply:
in another PR? because I don't see anything done about it in the PR here

Reply:
yes, when I will merge it to develop I will sync this branch